src/utils/offer/OfferUtils.js

```diff
@@ -15,3 +15,10 @@ const HumanReadableErrors = Object.freeze({
 });
 
 export const getHumanError = (error) => generalHumanError(error, HumanReadableErrors);
+
+export const validApplyURL = (val) => {
+    const httpRegex = /^https?:\S+$/;
```

Comment:
I used the simplified expression that's going to be used in the BE. However, it may be a little weird for the company to be typing the URL and easily see that `http:hi` is a valid URL for example. What do you think?

Reply:
I think that it's in the companies' best interest that this functionality works for them as they expect, so even though they would see an "incomplete" URL being marked as valid, it would be a minor detail as they would eventually supply a "complete" URL.

Reply:
[www.]*[chars].[chars]